### DISCUSSION TITLE
Date picker

### DISCUSSION THREAD

**Discussion by rdewolff** [tamagui/tamagui#590] [General]
Any plan to add a date picker to the lib? Would be amazing to have a nice date and time picker with tamagui 🙌

**Comment by rdewolff** [comment]
I found some effort done by @dohomi working on [tamagui-extra](https://www.npmjs.com/package/tamagui-extras)

**Comment by lobor** [comment]
I have created custom input date, work on browser and native
it is look like in my first work
![image](https://user-images.githubusercontent.com/3316940/216830867-c717895a-b5ea-4067-910d-f52993617879.png)

And now is it like
![image](https://user-images.githubusercontent.com/3316940/216831025-2b2a5c7d-fcfd-4ada-bf4b-db7039056227.png)
![image](https://user-images.githubusercontent.com/3316940/216831079-0ba52f9d-7367-4985-8b49-da5156e244ce.png)

**Reply by dohomi** [reply]
@lobor seems similar to https://tamagui-extras.vercel.app/?path=/story/ui-date-dateselection--single

**Reply by shahid-0** [reply]
@dohomi This package is no longer supported.

**Reply by dohomi** [reply]
https://tamagui-extras.vercel.app/?path=/story/date-date-selection--single Link seemed outdated
